src/event/eventPlugins/ResponderEventPlugin.js

```diff
@@ -570,7 +570,7 @@ var ResponderEventPlugin = {
 
   injection: {
     /**
-     * @param {{onChange: (ReactID, ReactID) => void} GlobalResponderHandler
+     * @param {{onChange: (ReactID, ReactID) => void}} GlobalResponderHandler
```

Comment:
I don't think an object spec like this is even valid jsdoc so it probably doesn't matter.